plugins/modules/zos_mvs_raw.py

```diff
@@ -1532,6 +1538,10 @@
 backups = []
 
 
+# Use of global tmphlq to keep coherent classes definitions
+tmphlq = ""
```

Comment:
global variable with the same name as a parameter?  I believe that is a bit unsafe.

Reply:
Have no specific reason for it to be the same name as a parameter. 

Stick with the use of global variable in this case because the tmphlq is used under build_data_definition to add it to the dd_data_set statements, however in this module we have the option of dd_concat which allows users multiple dd statements. 

Did not wanted to modify the interface of build_data_definition and build_dd_statements to pass it as parameter since conceptually does not make much sense. 

Another option is we could avoid using a global variable if we set the tmphlq for dd_data_set separately, could be another function called from main(). However we still need to set it for the VIODefinition class in the same build_data_definition function.

I'm still thinking global variable is the best option although it disturbs me a bit. 

With that said I'm open to suggestions on this.